tensorflow/lite/experimental/litert/vendors/qualcomm/core/utils/BUILD

```
@@ -0,0 +1,53 @@
+# Copyright (c) Qualcomm Innovation Center, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//tensorflow/lite/experimental/litert/build_common:litert_build_defs.bzl", "litert_test")
+
+package(
+    # copybara:uncomment default_applicable_licenses = ["//tensorflow:license"],
+    default_visibility = ["//tensorflow/lite/experimental/litert:__subpackages__"],
+)
+
+cc_library(
+    name = "log_utils",
```

Seems that people usually use the name without ".h" as target name.
```suggestion
    name = "log",
```